js/eventEmitter.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
```

Comment:
rather than exposing entire EventEmitter interface, just expose necessary methods...

module.exports = {
   emit: eventEmitter.emit,
   on: eventEmitter.on
}

also make clear in documentation that this can only be used main process.

i would like a better name but haven't thought of one yet.